cfe_internal/enterprise/federation/federation.cf

```diff
@@ -706,20 +706,28 @@ body contain cfpostgres_user
 
 bundle agent exported_data
 # @brief Run script to dump pg data on feeder hub
+# @note Since version 3.24.0, the inventory view is automatically refreshed by
+# cf-reactor whenever inventory is updated. Hence, you can assume that the
+# inventory is refreshed. However, there is one exception; if the previous
+# refresh happened too recently, the following one will be delayed.
 {
+@if before_version(3.24.0)
   methods:
     am_feeder.!am_paused::
       "Refresh Inventory"
         usebundle => "default:cfe_internal_refresh_inventory_view",
```

Comment:
Should I add a `action_policy` -> `warn` if the `cfe_internal_refresh_inventory_view` bundle is run on CFEngine >= 3.24.0 ?

Reply:
It shouldn't run at all if the binary is 3.24.0 or greater since it will be macrod out of scope. So, no, I would not do that.